Car/TeamNull SmartCar/src/util/HardwareUtil.cpp

```diff
@@ -77,11 +83,16 @@ void initialiseSensors() {
   delay(150);
   rearSensor.setTimeout(500);
   // rearSensor.setAddress((uint8_t)01);
-  if (!rearSensor.init()) {
+  while (!rearSensor.init()) {
     Serial.println("Failed to initialise REAR sensor");
-    blink(10);
-    while (1) {
-    }
+    blink(2);
+    pinMode(frontSensorPin, OUTPUT);
+    digitalWrite(frontSensorPin, LOW);
+
+    delay(1500);
+    digitalWrite(frontSensorPin, HIGH);
+    delay(200);
+    pinMode(frontSensorPin, INPUT);
```

Comment:
What does this (duplicated code) do?

Reply:
Yes, it's duplicated, will not be once we pass the debugging stage of trying to get two VL53L0Xs to work simultaneously.